components/settings/billing/tab-navigation.tsx

```
@@ -1,199 +1,209 @@
+					const isActive = activeTab === tab.id;
+
+					return (
+						<button
+							key={tab.id}
+							onClick={() => onTabChange(tab.id)}
+							className={`flex flex-col items-center gap-2 p-4 rounded-xl transition-all duration-200 ${
+								isActive
+									? 'bg-theme-primary-50 dark:bg-theme-primary-900/20 border-2 border-theme-primary-200 dark:border-theme-primary-700/50'
+									: 'bg-slate-50 dark:bg-slate-700/50 border-2 border-transparent hover:bg-slate-100 dark:hover:bg-slate-600/50'
+							}`}
+						>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)